srcpkgs/audiotube/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'audiotube'
+pkgname=audiotube
+version=22.04
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_PREFIX_PATH=/usr/lib/python3.10/site-packages/pybind11/share/cmake"
+hostmakedepends="extra-cmake-modules qt5-host-tools qt5-qmake"
+makedepends="qt5-devel kirigami2-devel ki18n-devel kcoreaddons-devel kcoreaddon\
+s kcrash-devel gettext-devel python3-pybind11 yt-dlp python3-ytmusicapi"
```

Comment:
```suggestion
makedepends="qt5-devel kirigami2-devel ki18n-devel kcoreaddons-devel kcoreaddons
 kcrash-devel gettext-devel python3-pybind11 yt-dlp python3-ytmusicapi"
```
splitting at the nearest word boundary is fine

Reply:
and you probably want kcoreaddons as a *host*makedepend, if it's trying to run things like `desktoptojson`